<ui version="4.0" >
 <class>AlignmentDlg</class>
 <widget class="QDialog" name="AlignmentDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Alignment parameters</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Translation</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Reference point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="referencePoint" >
        <item>
         <property name="text" >
          <string>Barycentre</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Bounding-box centre</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Rotation</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="noRotation" >
        <property name="text" >
         <string>None</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="twoPointsRotation" >
          <property name="text" >
           <string>Two points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="rotationPt1" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="rotationPt2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AlignmentDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlignmentDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twoPointsRotation</sender>
   <signal>toggled(bool)</signal>
   <receiver>rotationPt1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>100</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twoPointsRotation</sender>
   <signal>toggled(bool)</signal>
   <receiver>rotationPt2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>48</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel" >
     <x>405</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
